cpp/Platform.Setters/SetterBase.h

```diff
@@ -1,16 +1,16 @@
-﻿namespace Platform::Setters
-{
-    template <typename ...> class SetterBase;
-    template <typename TResult> class SetterBase<TResult> : public ISetter<TResult>
-    {
-        protected: TResult _result = 0;
-        
-        public: TResult Result() { return _result; }
-        
-        protected: SetterBase() { }
-    
-        protected: SetterBase(TResult defaultValue) { _result = defaultValue; }
-        
-        public: void Set(TResult value) { _result = value; }
-    };
-}
+﻿namespace Platform::Setters
+{
+    template <typename ...> class SetterBase;
+    template <typename TResult> class SetterBase<TResult>
+    {
+        protected: TResult _result {};
+
+        public: TResult Result() { return _result; }

```

Comment:
@Konard maybe use `const TResult&` to return type

Reply:
No, I don't think so. In the original was no ref return.

Reply:
If we return const reference then we will avoid copying when reading only. 
Else mark `Return()` as `[[nodiscard]]`

Reply:
At the moment, I think coping of value here is ok.